benchopt/plotting/generate_html.py

```diff
@@ -180,24 +180,16 @@ def shape_solvers_for_html(df, objective_column):
         df_filtered = df_filtered.replace([np.inf, -np.inf], np.nan)
         df_filtered = df_filtered.dropna(subset=[objective_column])
 
-        q1, q9 = compute_quantiles(df_filtered)
-        groupby_stop_val_median = df_filtered.groupby('stop_val').median()
-        color, marker = get_solver_style(solver)
-
-        stopping_strategy = df_filtered['stopping_strategy'].unique()
```

Comment:
to keep the code simple instead of adding a new function, what about
```
try:
    stopping_strategy = df_filtered['stopping_strategy'].unique()
except KeyError:
    stopping_strategy = ["Time"]
```

?

Reply:
Yes this seems much simpler and avoid making the code too complex.

Reply:
Fair enough, thanks